### DISCUSSION TITLE
Allow leading whitespace when rules "Append description with..."

### DISCUSSION THREAD

**Discussion by scottdk** [firefly-iii/firefly-iii#8191] [General]
I am adding information to a transaction's description with a rule. I can't find a way to add a SPACE to the beginning of the appended text. Is this possible to do?

![image](https://github.com/firefly-iii/firefly-iii/assets/11282782/fe0464f7-9a12-4d26-a408-9d256583b49a)

Original description: `Transfer from Dummy 3`
Desired description: `Transfer from Dummy 3 TO Dummy 1`

If I set `Append description with...` to ` TO Dummy 1`  (leading space, no quotes) I get:
`Transfer from Dummy 3TO Dummy 1`

If I add quotes to the append text `" TO Dummy 1"` (Leading space, text inside double quotes), the quotes (single or double) are included in the appended text:
`Transfer from Dummy 3" TO Dummy 1"`

**Comment by JC5** [comment]
I'll make sure the rule action includes some white space.
